File: 1011.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var var_Raio = parseFloat(lines.shift());
const CONST_PI = 3.14159
var var_Volume = (4.0 / 3) * CONST_PI * Math.pow(var_Raio, 3);

console.log("VOLUME = " + var_Volume.toFixed(3));

File: 1011.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double pi = 3.14159;
        double raio = double.Parse(Console.ReadLine());
        
        double volume = (4.0 / 3) * pi * Math.Pow(raio, 3);
        
        Console.WriteLine("VOLUME = " + volume.ToString("F3"));
    }
}